source/locationHelper.py

```diff
@@ -0,0 +1,402 @@
+#locationHelper.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2017 NV Access Limited, Babbage B.V.
+
+"""Classes and helper functions for working with rectangles and coordinates."""
+
+from collections import namedtuple
+import windowUtils
+import winUser
+from ctypes.wintypes import RECT, SMALL_RECT, POINT
+import textInfos
+import wx
+
+class Point(namedtuple("Point",("x","y"))):
+	"""Represents a point on the screen."""
+
+	def __add__(self,other):
+		if not isinstance(other,_POINT_CLASSES):
+			return NotImplemented

```

Comment:
Why do you not throw a NotImplementedError?

Reply:
See [this](https://stackoverflow.com/questions/878943/why-return-notimplemented-instead-of-raising-notimplementederror) on Stack Overflow